examples/mnist/train_mnist_custom_loop.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+from __future__ import print_function
+
+try:
```

Comment:
Do you use matplotlib in this code? As far as I could see, there is no part where matplotlib is used. This is different from `examples/mnist/train_mnist.py`, where matplotlib may be used in PlotReport.